test/system/005-info.bats

```diff
@@ -305,4 +305,27 @@ EOF
     run_podman $safe_opts system reset --force
 }
 
+@test "podman - empty string defaults for certain values" {
+    skip_if_remote "Test uses nonstandard paths for c/storage directories"
+
+    # We just want this to be empty - so graph driver will be set to the empty string
+    touch $PODMAN_TMPDIR/storage.conf
+
+    safe_opts=$(podman_isolation_opts ${PODMAN_TMPDIR})
+
+    # Force all custom directories so we don't pick up an existing database
+    CONTAINERS_STORAGE_CONF=$PODMAN_TMPDIR/storage.conf run_podman 0+w $safe_opts info
+    require_warning "The storage 'driver' option should be set" \
+       	            "c/storage should warn on empty storage driver"
+
+    # Now add a valid graph driver to storage.conf
+    cat >$PODMAN_TMPDIR/storage.conf <<EOF
+[storage]
+driver="overlay"
```

Comment:
```suggestion
driver="$(podman_storage_driver)"
```

That should use the right storage driver and turn into vfs on fedora -1, I am somewhat sure this is the *right* fix for the test instead of ignoring the vfs error